aibolit/utils/java_class_method.py

```diff
@@ -49,3 +50,8 @@ def name(self) -> str:
     @property
     def java_class(self) -> 'JavaClass':
         return self._java_class
+
+    def cfg(self) -> Graph:
```

Comment:
It might be a property, because we are not intended to change AST anyhow.
Also just noticed, we could use single `@cached_property` decorator from `functools` instead of composing `@property` decorator with `@lru_cache`. It will assume that cache is used only for one property.